src/inc/corcompile.h

```diff
@@ -727,6 +727,9 @@ enum CORCOMPILE_FIXUP_BLOB_KIND
     ENCODE_CHECK_FIELD_OFFSET,
 
     ENCODE_DELEGATE_CTOR,
+
+    ENCODE_GENERIC_DICTIONARY_ACCESS,
```

Comment:
It may be nice to add `ENCODE_DICTIONARY_LOOKUP_TYPE` and `ENCODE_DICTIONARY_LOOKUP_METHOD` aside from the rest since they are special prefixes - maybe put them at front and give them values 0x8 and 0x9.